<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginGui</class>
 <widget class="QWidget" name="PluginGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="mergeSheetsButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>merge sheets</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="ExQLineEdit" name="selectOpenFileEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>file to merge</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="viewInfoButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>view info</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="ExQLineEdit" name="selectSaveDirEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>directory to save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExQLineEdit</class>
   <extends>QLineEdit</extends>
   <header>exqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
